src/js/page/ui/material-slider.js

```diff
@@ -42,7 +42,6 @@ export default class MaterialSlider {
     this.range.classList.add('active');
 
     const upListener = () => {
-      // IE requires me to do this. Pah.
```

Comment:
I removed this comment because it appears to be incorrect; Without the next call, I saw breakage on at least Firefox.

Reply:
Hah IE was right! Yeah other browsers were getting the event loop wrong here.